SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -0,0 +1,113 @@
+/* global QUnit analyse */
+
+QUnit.test("Format preview test", function( assert ) {
+    let inLineTags = "[ibfg]|sc";
+    let suppress = {};
+    let text;
+    let issArray;
```

Comment:
could we pass this to issueTest each time instead of a sort of global?

Reply:
That would make it clearer. I have perhaps taken too literally Douglas Crockford's opinion that it is a "very good thing" that "inner functions get access to the variable of functions they are defined within".